src/Analyzer.Core.JsonRuleTests/README.md

```diff
@@ -0,0 +1,46 @@
+# Test Project for Built-in JSON Rules
+This project is designed for quickly adding tests to verify the correctness of built-in JSON rules.  When new rules are added, tests should be added here to make sure the rule is written correctly and help protect against incorrect changes to the rule.
+
+## Test Setup
+A test (or tests) for a given JSON rule consists of 2 parts:
+1. A JSON configuration describing the test(s).
+2. One or more test ARM templates to analyze.  The results of the analysis are compared with the test configuration to assert correctness of the JSON rule.
+
+### JSON Configuration
+To create a set of tests for a JSON rule, a new JSON file is created in the *Tests* directory.  The name of the file must be the same as the `name` property of the JSON rule, with ".json" as the file extension.
```

Comment:
"The name of the file must be the same as the `name` property of the JSON rule" should probably be bold

Reply:
Is there a reason the template field doesn't include the file extension? I feel like it would be easier to follow if it did.

Reply:
Not including the extension in the Template field allows enforcement of the '.badtemplate' extension.  Whether we keep that requirement is still up for discussion though.